src/extractor/myexperience/posting/status.ts

```diff
@@ -0,0 +1,76 @@
+export type JobPostingStatus = "APPROVED" | "EXPIRED" | "UNKNOWN";
+
+export type InternalStatus =
+  | "FILLED"
+  | "PARTIALLY-FILLED"
+  | "UNFILLED"
+  | "EMP-RANKINGS-FINALIZED"
+  | "INTERVIEW-COMPLETE"
+  | "INTERVIEW-SELECTIONS-COMPLETE"
+  | "EXPIRED-APPLICATIONS-AVAILABLE"
+  | "OPEN-FOR-APPLICATIONS"
+  | "CANCEL"
+  | "UNKNOWN";
+
+export type ApplicationStatus =
+  | "APPLIED"
+  | "SHORTLISTED"
+  | "NOT-INTERESTED"
+  | "EXPIRED"
+  | "AVAILABLE"
+  | "UNKNOWN";
+
+export type InterviewStatus =
+  | "APPLIED"
+  | "NOT-SELECTED"
+  | "SELECTED-FOR-INTERVIEW"
+  | "EMPLOYED"
+  | "NONE"
+  | "UNKNOWN";
```

Comment:
Are all of these exactly the same on SFU?

Reply:
Do people actually use the internal status for Waterloo? Almost no one does for SFU so I don't actually know what all the tags are. I wasn't even sure if they were standardized.

Looking though the available jobs right now, the ones I see are "Interviewing Phase", "Filled externally" and "Offer Phase" (only one instance of each) so I guess I'll add those (minus filled externally which is already covered by filled) and probably leave the rest as is.

Applied is "Application Submitted" so I'll add that and Job Posting status is the same.

Reply:
Yes, the internal status is used often for jobs on WW. "Filled externally" can be its own status.

A lot of these aren't as applicable to SFU, but since there is a separate function to parse the status, you can add them retroactively if you find edge cases. You can remove the ones which you can't find examples for.